src/verify_ce_config.py

```diff
@@ -59,19 +59,22 @@ def main():
         try:
             config_val = htcondor.param[attr]
         except KeyError:
-            error("Missing required %s configuration value" % attr)
+            warn("Missing %s configuration value. " % attr +
+                 "Can be safely ignored if using 'JOB_ROUTER_ROUTE_*' ClassAd transforms.")
 
         # store the ads (iterating through ClassAdStringIterator consumes them)
         try:
             parsed_jr_ads[attr] = list(classad.parseAds(config_val))
         except ValueError:
             # We shouldn't ever get here since classad.parseAds() only raises ValueError when it's given
```

Comment:
"We shouldn't ever get here" implies at least an error, if not an assertion.

Reply:
Agreed but I'm not keen on keeping this an error that could prevent CE startup without knowing more about how an unset `JOB_ROUTER_ENTRIES` interacts with this piece of code.

Reply:
Can you update the comment then?

Reply:
Actually, now that KeyError doesn't exit (?), is it still true that we shouldn't get to the ValueError exception?
Or, worse, can
```
            parsed_jr_ads[attr] = list(classad.parseAds(config_val))
```
crash?

Reply:
D'oh, good point